example/lib/main.dart

```diff
@@ -24,16 +24,19 @@ class _MyAppState extends State<MyApp> {
 
   @override
   Widget build(BuildContext context) {
-    final colors = [
-      SystemTheme.accentInstance.lightest,
-      SystemTheme.accentInstance.lighter,
-      SystemTheme.accentInstance.light,
-      SystemTheme.accentInstance.accent,
-      SystemTheme.accentInstance.dark,
-      SystemTheme.accentInstance.darker,
-      SystemTheme.accentInstance.darkest,
-    ];
-    return Scaffold(
+
+    return SystemThemedWidget(
```

Comment:
Where is this widget from?

Reply:
sorry, system_theme_widget.dart was skipped in previous commits